doc/user/olm-catalog/operator-bundles.md

```diff
@@ -0,0 +1,23 @@
+# Operator Bundles
+
+This document gives an overview of using `operator-sdk` to work with Operator [bundles][registry-bundle], namely on-disk bundle directories and creating bundle [images/metadata][registry-bundle-image].
+
+## Commands
+
+The following `operator-sdk` subcommands create or interact with Operator on-disk bundles and bundle images:
+
+* `generate csv`: creates a new or updates an existing CSV in a semantically versioned bundle directory, creates a package manifest if it does not exist, and optionally copies your CRDs to the versioned bundle directory. Read more about this command [here][sdk-generate-csv].
+* `bundle create`: creates an Operator bundle image from manifests on disk, or writes bundle image metadata to disk. This subcommand is effectively the same command as `opm alpha bundle build`. Output and underlying behavior between these commands is the same, except nothing will be written to disk if `--generate-only` is unset (the default). Refer to [`opm alpha bundle build` docs][registry-opm-build] for more information. There are a few CLI differences between these commands:
```

Comment:
Do you need to add a reference to `opm alpha bundle build`? Is it still in the project? If not, could we not just drop it?
Also, if has the bundle to do the same way we are keeping these alpha ones? Should not they be removed? 

Reason: for users, it may Shows confuse. E.g Why have 2 commands to do the same? What means alpha? and etc... 
In this way, IHMO we should just doc the bundle commands. Also, if we replace the alpha commands by these ones then we should remove/deprecated the alpha ones.

Reply:
`opm` will continue to support these commands AFAIK. I'd rather not duplicate documentation to avoid drift. The corresponding commands in `operator-sdk` are not in alpha. The wording could use some work to indicate this.